pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -51,7 +51,7 @@ let
 
     AR = "${stdenv'.cc.targetPrefix or ""}ar";
     CC = "${stdenv'.cc.targetPrefix or ""}cc";
-    CPP = if (stdenv'.cc.isClang or false) then "clang-cpp" else "cpp";
+    CPP = "${stdenv'.cc.targetPrefix or ""}cpp";
```

Comment:
@ericson2314 should cpp be prefixed? I feel like it is one case where we don't want the prefixed one. Everyone target could use the same cpp.

Reply:
I don't know much about cross stuff, but I think cpp knows about the target to be able to define macros like `__i386__` and such (e.g. https://github.com/openbsd/src/blob/master/libexec/tradcpp/config.h)

Reply:
It's just safer to prefix